service/src/test/java/uk/gov/hmcts/reform/fpl/service/ApplicantServiceTest.java

```diff
@@ -47,8 +51,9 @@ void shouldNotExpandApplicantCollectionWhenApplicantsAlreadyExists() {
                 .build()))
             .build();
 
-        assertThat(service.expandApplicantCollection(caseData)).hasSize(1);
-        assertThat(service.expandApplicantCollection(caseData).get(0).getValue().getParty().partyId).isEqualTo(uuid);
+        assertThat(service.expandApplicantCollection(caseData, organisation)).hasSize(1);
+        assertThat(service.expandApplicantCollection(caseData, organisation)
```

Comment:
we could probably use ElementUtils.unwrapElements method here to make this a little simpler

Reply:
done :)